src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -729,6 +729,7 @@ private function rowToArray(Row $row, $locale, $webspaceKey, $fields, $onlyPubli
             'changed' => $document->getChanged(),
             'changer' => $document->getChanger(),
             'created' => $document->getCreated(),
+            'publishedState' => $nodeState === WorkflowStage::PUBLISHED,
```

Comment:
Why did you have to touch this file? Is there still a dependency to it in the scope of this PR?